modules/NuGetPackageVerifier/console/Rules/AssemblyIsBuiltInReleaseConfigurationRule.cs

```diff
@@ -20,47 +20,70 @@ public IEnumerable<PackageVerifierIssue> Validate(PackageAnalysisContext context
+                {
+                    // packageFileStream is not a seekable stream. So wrap it in a memory stream for PEReader to consume.
+                    packageFileStream.CopyTo(stream);
+                    stream.Position = 0;
+                    PEReader peReader = null;
+                    MetadataReader reader;
                     try
                     {
-                        using (var packageFileStream = context.PackageReader.GetStream(currentFile))
-                        using (var stream = new MemoryStream())
+                        peReader = new PEReader(stream);
+                        reader = peReader.GetMetadataReader();
+                    }
+                    catch
+                    {
+                        // This assembly is not managed.
+                        peReader?.Dispose();
+                        continue;
+                    }
+
+                    using (peReader)
+                    {
+                        foreach (var handle in reader.CustomAttributes)
                         {
-                            // packageFileStream is not a seekable stream. So wrap it in a memory stream for PEReader to consume.
-                            packageFileStream.CopyTo(stream);
-                            stream.Position = 0;
-                            using (var peReader = new PEReader(stream))
+                            var customAttribute = reader.GetCustomAttribute(handle);
+                            string typeName;
+                            if (customAttribute.Constructor.Kind == HandleKind.MemberReference)
                             {
-                                var reader = peReader.GetMetadataReader();
-                                foreach (var handle in reader.CustomAttributes)
-                                {
-                                    var customAttribute = reader.GetCustomAttribute(handle);
-                                    var ctor = reader.GetMemberReference((MemberReferenceHandle)customAttribute.Constructor);
-                                    var type = reader.GetTypeReference((TypeReferenceHandle)ctor.Parent);
-                                    var typeName = reader.GetString(type.Name);
-                                    if (string.Equals(typeof(DebuggableAttribute).Name, typeName, StringComparison.Ordinal))
-                                    {
-                                        var attribute = customAttribute.DecodeValue(NullProvider.Instance);
+                                var ctor = reader.GetMemberReference((MemberReferenceHandle)customAttribute.Constructor);
+                                var type = reader.GetTypeReference((TypeReferenceHandle)ctor.Parent);
+                                typeName = reader.GetString(type.Name);
+                            }
+                            else if (customAttribute.Constructor.Kind == HandleKind.MethodDefinition)
```

Comment:
Need to figure out why this doesn't happen on the CI. Locally running this on the signed package list was failing for the test specification package when it encountered `NotNullAttribute`